### PR TITLE
[FEAT] delete kutipan UI

### PR DESCRIPTION
**Pull request by adamjanin**
ScreenRecording2024-04-15at9 33 04AM-ezgif com-optimize

logs :
2024-04-15 09:33:11 [GIN] 2024/04/15 - 01:33:11 | 204 |     107.708µs |    192.168.65.1 | OPTIONS  "/kutipan/12"
2024-04-15 09:33:11 {"level":"info","time":"2024-04-15T01:33:11Z","message":"delete kutipan by id : 12"}
2024-04-15 09:33:11 [GIN] 2024/04/15 - 01:33:11 | 200 |    8.798291ms |    192.168.65.1 | DELETE   "/kutipan/12"
2024-04-15 09:33:19 [GIN] 2024/04/15 - 01:33:19 | 204 |       44.75µs |    192.168.65.1 | OPTIONS  "/kutipan/16"
2024-04-15 09:33:19 {"level":"info","time":"2024-04-15T01:33:19Z","message":"delete kutipan by id : 16"}
2024-04-15 09:33:19 [GIN] 2024/04/15 - 01:33:19 | 200 |    4.606125ms |    192.168.65.1 | DELETE   "/kutipan/16"

### CODE REVIEW

File: dashboard/src/views/tabung/cetak/Cetak.js [hunk 1]
```diff
@@ -12,13 +12,13 @@ import {
...
   CForm,
-  CFormInput,
+  CFormInput, CModalHeader, CModalTitle
 
 } from '@coreui/react'
 import { getKutipan } from 'src/service/tabung/KutipanApi'
 import { getTabung } from 'src/service/tabung/TabungApi'
 import DataTable from 'react-data-table-component'
-import { cilInfo, cilPrint, cilPencil } from '@coreui/icons'
+import { cilInfo, cilPrint, cilPencil, cilBasket } from '@coreui/icons'
```
Comment (rohaizanr) [inline dashboard/src/views/tabung/cetak/Cetak.js:21]: icon delete takde eh ?

**Review by rohaizanr — APPROVED** [review]

File: dashboard/src/views/tabung/cetak/Cetak.js (hunk 1 above)
Comment (rohaizanr) [reply dashboard/src/views/tabung/cetak/Cetak.js:21]: @adamjanin cil-trash ok kot

Comment (adamjanin) [reply dashboard/src/views/tabung/cetak/Cetak.js]: okay, fixed. haha, tak perasan lak ada icon untuk trash. 

image

**Review by rohaizanr — APPROVED** [review]
terima kasih

**Pull request merged**